specs/data_structures.md

```diff
@@ -114,36 +132,32 @@ Data that is [erasure-coded](#erasure-coding) for [data availability checks](htt
 
 | name         | type                        | description |
 | ------------ | --------------------------- | ----------- |
-| `height`     | `uint64`                    |             |
-| `round`      | `uint64`                    |             |
+| `height`     | [Height](#type-aliases)     |             |
+| `round`      | [Round](#type-aliases)      |             |
 | `blockID`    | [BlockID](#blockid)         |             |
 | `signatures` | [CommitSig](#commitsig)`[]` |             |
 
-## Time
+## Timestamp
+
+Timestamp is a [type alias](#type-aliases).
 
 LazyLedger uses a 64-bit unsigned integer (`uint64`) to represent time in [TAI64](http://cr.yp.to/libtai/tai64.html) format.
 
 ## BlockID
 
-The block ID is a single Merkle root: the root of the [block header](#header)'s fields, in the order provided in the spec. The root is computed using a [binary Merkle tree](#binary-merkle-tree).
+BlockID is a [type alias](#type-aliases).
 
-| name         | type                      | description                      |
-| ------------ | ------------------------- | -------------------------------- |
-| `headerRoot` | [HashDigest](#hashdigest) | Root of the block header fields. |
+The block ID is a single Merkle root: the root of the [block header](#header)'s fields, in the order provided in the spec. The root is computed using a [binary Merkle tree](#binary-merkle-tree).
```

Comment:
BTW (unrelated to this PR): the binary tree also needs to be explicit about (leaf/inner node) domain separation. That's prob. already captured in the issue about the merkle tree section anyways.